src/Core/StaticConfigTrait.php

```diff
@@ -36,6 +34,14 @@
     protected static $_config = [];
 
     /**
+     * An array mapping url schemes to fully qualified caching engine
```

Comment:
It's not just for `caching engine`. The mapping can be potentially used any class using this trait.